src/custom/components/Tooltip/TooltipMod.tsx

```diff
@@ -7,6 +7,7 @@ const TooltipContainer = styled.div`
   padding: 0.6rem 1rem;
   font-weight: 400;
   word-break: break-word;
+  font-size: smaller;
```

Comment:
I keep butchering your `smaller` font-size definitions as IMO it's better to use a specific PX size. Untill we have a typography theme in place (e.g. theme.fontSmall).

Reply:
@biocom please make an issue for these and we can address them post merge, thanks!!